pointer.go

```diff
@@ -17,13 +17,13 @@ func Save(v interface{}) unsafe.Pointer {
 		return nil
 	}
 
-	// Generate real fake C pointer.
-	// This pointer will not store any data, but will bi used for indexing purposes.
-	// Since Go doest allow to cast dangling pointer to unsafe.Pointer, we do rally allocate one byte.
-	// Why we need indexing, because Go doest allow C code to store pointers to Go data.
+	// Generates a real fake C pointer.
+	// The pointer won't store any data but be used for indexing purposes.
+	// As Go doesn't allow to cast a dangling pointer to "unsafe.Pointer", we do really allocate one byte.
+	// Indexing is needed because Go doesn't allow C code to store pointers to Go data.
```

Comment:
// Indexing is necessary because Go doesn't allow C code to store pointers to Go data.